.eslintrc.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+    "env": {
+        "browser": true,
+        "commonjs": true,
+        "es6": true,
+        "node": true
+    },
+    "extends": "eslint:recommended",
+    "parserOptions": {
+        "ecmaVersion": 2016
+    },
+    "rules": {
+        "indent": [
```

Comment:
We need to remove the indent rule, we want to ignore file endings at the moment, we work in windows and linux, git will control file endings for us.

Reply:
I do have a question, in the Issue #16  it mentions to use tabs for indent and not spaces.

Reply:
I meant the lineEndings rule, i apologize, not the indent rule.